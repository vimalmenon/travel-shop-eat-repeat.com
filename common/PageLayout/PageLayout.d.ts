import { IMetaData } from '@types';

export interface IPageLayoutProps {
  metaData: IMetaData;
  navigationList: Record<string, IMetaData[]>;
}
