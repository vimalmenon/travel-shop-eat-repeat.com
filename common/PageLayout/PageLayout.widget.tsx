import { Footer } from './Footer';
import { Header } from './Header';

export const PageLayout: React.FC = ({ children }) => {
  return (
    <main className="flex flex-col">
      <Header />
      <section className="flex text-sm py-1 px-2">{children}</section>
      <Footer />
    </main>
  );
};
