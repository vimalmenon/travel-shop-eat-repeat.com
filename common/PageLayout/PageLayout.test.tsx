import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { PageLayout } from './PageLayout.widget';

describe('common > <PageLayout> Component', () => {
  describe('Snapshot Testing', () => {
    it('<PageLayout>', () => {
      const wrapper = shallow(<PageLayout />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
