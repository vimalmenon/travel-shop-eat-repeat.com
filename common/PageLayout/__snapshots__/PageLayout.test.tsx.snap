// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common > <PageLayout> Component Snapshot Testing <PageLayout> 1`] = `
<main
  className="flex flex-col"
>
  <Header />
  <section
    className="flex text-sm py-1 px-2"
  />
  <Footer />
</main>
`;
